ecosystem/indexer-grpc/indexer-grpc-cache-worker/src/worker.rs

```diff
@@ -323,7 +334,8 @@ async fn process_streaming_response(
     }
     let mut current_version = starting_version;
     let mut starting_time = std::time::Instant::now();
-
+    let mut last_file_update_check_timestamp =
+        std::time::Instant::now() - std::time::Duration::from_secs(GCS_LOOKUP_FREQUENCY_IN_SECS);
```

Comment:
nit: can we just set this to `std::time::Instant::now()` for simplicity's sake

Reply:
done.